CHANGELOG.md

```diff
@@ -0,0 +1,51 @@
+# Changelog
+
+## [master]
+
+[master]: https://github.com/DripEmail/drip-ruby/compare/v2.0.0...HEAD
+
+* Your contribution here!
+
+## [`2.0.0`] (2018-03-27)
+
+[`2.0.0`]: https://github.com/DripEmail/drip-ruby/compare/v1.0.0...v2.0.0
```

Comment:
Could also link to the release page.  And then on the release description you could add the link that shows the compare if you'd like.